taxonomies/brands.txt

```diff
@@ -238,3 +244,2494 @@ wikidata:en: Q10724505
+wikidata:en: Q98754911
+
+xx: Whisps
+wikidata:en: Q99201392
+
+xx: Town House Crackers
+wikidata:en: Q99434048
+
+xx: Better Than Bouillon
+wikidata:en: Q99509957
+
+xx: Classico
+wikidata:en: Q99522861
+
+xx: Duncan Hines
+wikidata:en: Q99658929
+
+xx: Barratt Sweets
+wikidata:en: Q99906120
+
+xx: Incogmeato
+wikidata:en: Q100156149
+
+xx: Lemonilo
+wikidata:en: Q101432536
+
+xx: NotCo
+wikidata:en: Q101609467
+
+xx: Chocoramo
+wikidata:en: Q103827445
+
+xx: Wensleydale Creamery
+wikidata:en: Q104841912
+
+xx: My/Mo Mochi Ice Cream
+wikidata:en: Q104844654
+
+xx: Rally
+wikidata:en: Q104867091
+
+xx: Just Egg
+wikidata:en: Q105072445
+
+xx: Just Egg Folded
+wikidata:en: Q105072472
+
+xx: Aztec
+wikidata:en: Q105637554
+
+xx: Dinty Moore
+wikidata:en: Q105758244
+
+xx: Mtn Dew Rise Energy
+wikidata:en: Q106107395
+
+xx: Fresh Express Incorporated
+wikidata:en: Q106490016
+
+xx: Fandangos
+wikidata:en: Q106645447
+
+xx: Colliders
+wikidata:en: Q106919072
+
+xx: Takis
+wikidata:en: Q107071944
+
+xx: Papa Rafkin
+wikidata:en: Q107353507
+
+xx: Wiener Zucker
+wikidata:en: Q108067972
+
+xx: Hedgehog flavour crisps
+wikidata:en: Q108179212
+
+xx: Chocolate Aguila
+wikidata:en: Q108190515
+
+xx: Terrabusi
+wikidata:en: Q108197751
+
+xx: Chio Chips
+wikidata:en: Q108201931
+
+xx: Edita
+wikidata:en: Q108248560
+
+xx: Egypt Foods Group
+wikidata:en: Q108489358
+
+xx: CurderBurger
+wikidata:en: Q108898648
+
+xx: Oatmeal Creme Pie
+wikidata:en: Q109058076
+
+xx: Barratt
+wikidata:en: Q109229390
+
+xx: Boltsi
+wikidata:en: Q109266587
+
+xx: Chien Tu Hot Pot
+wikidata:en: Q109344533
+
+xx: Abbey Crunch
+wikidata:en: Q109535897
+
+xx: Toffo
+wikidata:en: Q109558143
+
+xx: Cabana
+wikidata:en: Q109567454
+
+xx: Brussels Ketjep Brava
+wikidata:en: Q109914319
+
+xx: Brussels Ketjep
+wikidata:en: Q109914347
+
+xx: Brussels Ketjep Mayo
+wikidata:en: Q109914368
+
+xx: Brussels Ketjep Dallas
+wikidata:en: Q109914388
+
+xx: Brussels Ketjep Cowboy
+wikidata:en: Q109914567
+
+xx: Brussels Ketjep Mostoed
+wikidata:en: Q109914580
+
+xx: Plazma
+wikidata:en: Q110047332
+
+xx: Chococo
+wikidata:en: Q111631099
+
+xx: Kame Manjū
+wikidata:en: Q112053209
+
+xx: Stiksy
+wikidata:en: Q112672127
+
+xx: Jacky
+wikidata:en: Q112813989
+
+xx: Orange Bakery
+wikidata:en: Q113023049
+
+xx: Wybert
+wikidata:en: Q113077616
+
+xx: Pink Sauce
+wikidata:en: Q113466947
+
+xx: Magic Spoon
+wikidata:en: Q113489440
+
+xx: Chocolat Kohler
+wikidata:en: Q113624924
+
+xx: Wolf Nudeln
+wikidata:en: Q113633712
+
+xx: Tonicorp
+wikidata:en: Q114086238
+
+xx: Paqui
+wikidata:en: Q114397492
+
+xx: Pillsbury holiday sugar cookies
+wikidata:en: Q114779420
+
+xx: Danimals
+wikidata:en: Q115478274
+
+xx: Chicago Town
+wikidata:en: Q116168676
+
+xx: Hale's Blue Boy
+wikidata:en: Q116457216
+
+xx: Brussels Ketjep Andalose
+wikidata:en: Q116487533
+
+xx: Kofila
+wikidata:en: Q116601648
+
+xx: Biscoff
+wikidata:en: Q116641741
+
+xx: Lotus Biscoff Cookie Butter Spread
+wikidata:en: Q116642666
```

Comment:
Is this really what we understand as a brand in OFF? I was trying to compare with actual products, but… https://world.openfoodfacts.org/cgi/search.pl?search_terms=Lotus+Biscoff+Cookie+Butter+Spread

https://world.openfoodfacts.org/cgi/search.pl?search_terms=Biscoff+Cookie+Butter&search_simple=1&action=process did give some results though, but none of those look like they’d have a “Lotus Biscoff Cookie Butter Spread” brand.